docs/templates/properties.js

```diff
@@ -39,6 +39,20 @@ module.exports['has-errors'] = function ({ item }) {
   return item.inheritance.some(c => c.errors.length > 0);
 };
 
+module.exports.functions = function ({ item }) {
+  return [...findAll(['VariableDeclaration', 'FunctionDefinition'], item)].filter(f =>
+    f.nodeType === 'VariableDeclaration' ? f.visibility === 'public' : f.visibility !== 'private',
+  );
+};
+
+module.exports.returns2 = function ({ item }) {
```

Comment:
Why not `returnsExtended` or similar? I don't think `returns2` is really expressive

Reply:
I prefer the short version, if you feel strongly about it we could use `returnsExtended`.

Reply:
Let's keep `returns2`, don't want to block this